front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineController.groovy

```diff
@@ -152,4 +169,8 @@ class PipelineController {
 
   @InheritConstructors
   static class InvalidPipelineDefinition extends RuntimeException {}
+
+  @InheritConstructors
+  @ResponseStatus(value = HttpStatus.BAD_REQUEST, reason = "Provided ID doesn't match pipeline")
+  class InvalidPipelineRequestException extends RuntimeException {}
```

Comment:
Might be worth having an `@ExceptionHandler` that outputs the standard error response with a legitimate error message rather than the default `Provided ID doesn't match pipeline` message